packages/feedback/feedback.scss

```diff
@@ -45,3 +49,67 @@ $max-feedback-input-width: 30rem;
         animation: show 0.25s ease-out;
     }
 }
+
+.jkl-feedback-smiley-option {
+    display: inline-block;
+    position: relative;
+    cursor: pointer;
+    height: $smiley-size;
+    width: $smiley-size;
+    color: var(--jkl-feedback-smiley-color);
+    transform: translate3d(0, 0, 0);
+
+    @include motion("standard");
+    transition-property: transform, color;
+    will-change: transform, color;
```

Comment:
> Warning: will-change is intended to be used as a last resort, in order to try to deal with existing performance problems. It should not be used to anticipate performance problems. ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change))

Er det et behov for denne her?

Reply:
Hmm.. Har lært at den er fin å bruke hvis man er nødt til å endre verdier som vanligvis tvinger repaint av hele siden (`color`, i dette tilfellet), ettersom den tvinger innholdet over på et eget layer. Men det virker ikke som MDN er helt enige i det 😅 Har ikke sett på preformance med og uten i dette tilfellet, så kan godt hende det er bedre å la være!

Reply:
Fjerna det 😊 Så ikke ut til å ha så mye å si på min maskin, og tror vi gjør tyngre ting enn å animere farge ut en å bry oss andre steder!